### PR TITLE
Link speed for windows network fingerprinting

### PR DESCRIPTION
**Pull request by mwieczorek**
On windows it's always set to default 100Mbps
I think we can use standard powershell cmdlts to get the link speed
Let me know what you think.

### CODE REVIEW

**Comment by diptanu** [comment]
@mwieczorek Did you test this on Windows 8 and 10?

**Comment by mwieczorek** [comment]
Yes. On Windows 8.1, 10, and Windows Server 2016 TP5

File: client/fingerprint/network_windows.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	command := fmt.Sprintf("Get-NetAdapter -IncludeHidden | Where name -eq '%s' | Select -ExpandProperty LinkSpeed", device)
+	path := "powershell.exe"
+	outBytes, err := exec.Command(path, command).Output()
+
+	if err != nil {
+		f.logger.Printf("[WARN] fingerprint.network: Error calling %s (%s): %v", path, command, err)
+		return 0
+	}
+
+	output := strings.TrimSpace(string(outBytes))
```
Comment (dadgar) [inline client/fingerprint/network_windows.go]: I would pull everything from here down into a parse method and then unit test this method.

**Comment by dadgar** [comment]
LGTM

**Pull request merged**